src/App.tsx

```diff
@@ -1,223 +1,197 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  FormEvent,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  createTodo, deleteTodo, getTodos, updateTodo,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from
+  './components/Auth/ErrorNotification/ErrorNotification';
+import { getFilteredTodo } from './components/Auth/FilterTodos/getFilteredTodo';
+import { Footer } from './components/Auth/Footer/Footer';
+import { Header } from './components/Auth/Header/Header';
+import { TodoList } from './components/Auth/TodoList/TodoList';
+import { ErrorMessage } from './types/Error';
+import { FilterType } from './types/FilterBy';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.All);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [title, setTitle] = useState('');
+  const [selectedIds, setSelectedIds] = useState<number[]>([]);
+  const [isAdding, setIsAdding] = useState(false);
+  const [completedTodosId, setCompletedTodosId] = useState<number[]>([]);
+  const [toggle, setToggle] = useState<boolean>(true);
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
 
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
-    if (newTodoField.current) {
-      newTodoField.current.focus();
+    const getTodosFromServer = async (userId: number) => {
+      try {
+        const receivedTodos = await getTodos(userId);
+
+        setTodos(receivedTodos);
+      } catch (error) {
+        setErrorMessage(`${error}`);
+      }
+    };
+
+    if (!user) {
+      return;
     }
+
+    getTodosFromServer(user.id);
   }, []);
 
+  const filterTodoBy = useMemo(() => getFilteredTodo(filterType, todos),
+    [todos, filterType]);
+
+  const newTodo = useCallback(async (event: FormEvent) => {
```

Comment:
can you move it to Header? App will be easier )

Reply:
> Good work 👏. You can make your App file easier and I believe that you do it )

Done!